symbol-overlay.el

```diff
@@ -156,6 +156,12 @@
   :group 'symbol-overlay
   :type 'float)
 
+(defcustom symbol-overlay-cursor-hook
```

Comment:
This isn't a hook: hooks are lists of functions. This is just a function, so it should be called something like `symbol-overlay-cursor-sensor-function`.